### PR TITLE
Moved all mass boundary conditions to common namespace

### PR DESCRIPTION
**Pull request by isriva**
Moved all mass boundary conditions to common namespace. Also, modified the input parameters to use bc_Yk_{x,y,z}_{lo,hi} to provide the mass boundary conditions.

### CODE REVIEW

**Comment by isriva** [comment]
Andy, I tested on the "inputs_regression_noslip_reservoir_isothermal", and the code compiles and works.

File: src_common/InhomogeneousBCVal.H [hunk 1]
```diff
@@ -19,17 +19,17 @@ Real InhomogeneousBCVal ( int comp,
...
 
         if (x == prob_lo[0]) {
-            return c_bc_x_lo[comp-SPEC_BC_COMP];
+            return bc_Yk_x_lo[comp-SPEC_BC_COMP];
         } else if (x == prob_hi[0]) {
-            return c_bc_x_hi[comp-SPEC_BC_COMP];
+            return bc_Yk_x_hi[comp-SPEC_BC_COMP];
         } else if (y == prob_lo[1]) {
-            return c_bc_x_lo[comp-SPEC_BC_COMP];
+            return bc_Yk_x_lo[comp-SPEC_BC_COMP];
```
Comment (ajnonaka) [inline src_common/InhomogeneousBCVal.H:26]:
```suggestion
            return bc_Yk_y_lo[comp-SPEC_BC_COMP];
```

File: src_common/InhomogeneousBCVal.H [hunk 2]
```diff
@@ -19,17 +19,17 @@ Real InhomogeneousBCVal ( int comp,
...
         } else if (y == prob_hi[1]) {
-            return c_bc_x_lo[comp-SPEC_BC_COMP];
+            return bc_Yk_x_lo[comp-SPEC_BC_COMP];
```
Comment (ajnonaka) [inline src_common/InhomogeneousBCVal.H:28]:
```suggestion
            return bc_Yk_y_hi[comp-SPEC_BC_COMP];
```

File: src_common/InhomogeneousBCVal.H [hunk 3]
```diff
@@ -19,17 +19,17 @@ Real InhomogeneousBCVal ( int comp,
...
         } else if (z == prob_lo[2]) {
-            return c_bc_x_lo[comp-SPEC_BC_COMP];
+            return bc_Yk_x_lo[comp-SPEC_BC_COMP];
```
Comment (ajnonaka) [inline src_common/InhomogeneousBCVal.H:30]:
```suggestion
            return bc_Yk_z_lo[comp-SPEC_BC_COMP];
```

File: src_common/InhomogeneousBCVal.H [hunk 4]
```diff
@@ -19,17 +19,17 @@ Real InhomogeneousBCVal ( int comp,
...
         } else if (z == prob_hi[2]) {
-            return c_bc_x_lo[comp-SPEC_BC_COMP];
+            return bc_Yk_x_lo[comp-SPEC_BC_COMP];
```
Comment (ajnonaka) [inline src_common/InhomogeneousBCVal.H:32]:
```suggestion
            return bc_Yk_z_hi[comp-SPEC_BC_COMP];
```

**Pull request merged**